semantic_conventions/resource/redis.yaml

```
@@ -0,0 +1,11 @@
+groups:
+  - id: redis
+    prefix: redis
+    brief: >
+        A redis instance
+    attributes:
+      - id: instance
+        type: string
+        brief: >
+          Reported name of the redis instance
```

```suggestion
          Reported name of the Redis instance. This can be in the form `{host}:{port}` or any other name provided manually when setting up the instrumentation.
```
Not yet happy with my suggestion but this lacks clarification. I'm open for better suggestions :)

---

Done, added an appropriate description.